README.md

```diff
@@ -134,38 +134,51 @@ access_token = OAuth::AccessToken.new(QB_OAUTH_CONSUMER, string_access_token_fro
 Most likely you will want to persist the OAuth access credentials so you don't have to connect to QBO
 each and every time.
 
-QBO allows the access credentials to live for 6 months, and after 5 months they can be renewed. You will
-get an error if you try and renew prior to 5 months. Thus, you will need to keep track of the dates and
-manage the renewal process yourself.
+The access credentials are valid for 180 days. 30 days before they expire, they can be renewed. You'll need
+to keep track of the expiration date and manage the renewal process yourself.
 
 An example database table would have fields likes:
 
-
 ```sql
 access_token varchar(255),
-access_secret varchar(255)
+access_secret varchar(255),
```

Comment:
Yeah agreed that it would be too confusing to include encryption in the example. I'll add a note about the need to encrypt though.